lib/synth-audio.js

```diff
@@ -619,6 +619,8 @@ const synthElevenlabs = async(logger, {credentials, options, stats, language, vo
   if (!process.env.JAMBONES_DISABLE_TTS_STREAMING && !renderForCaching) {
     let params = '';
     params += `{api_key=${api_key}`;
+    params += ',vendor=elevenlabs';
+    params += ',voice=${voice}';
```

Comment:
I dont understand this...elevenlabs uses model_id instead of voice doesn't it?

Reply:
NO elevenlabs has 
voice= pNInz6obpgDQGcFmaJgB
language=en
model=turbo_v2